src/ussd_pmtct_seed.js

```diff
@@ -0,0 +1,583 @@
+                    }
+                }
+            });
+        });
+
+        self.add("state_end_optout", function(name) {
+            return new EndState(name, {
+                text: $(
+                    "You will not receive SMSs about keeping your baby HIV " +
+                    "negative. You will still receive MomConnect SMSs. To stop " +
+                    "receiving these SMSs, dial {{ mc_optout_number }}")
+                    .context({ mc_optout_number: self.im.config.mc_optout_channel }),
+                next: "state_start"
+            });
+        });
+
+        self.add("state_loss_messages", function(name) {
+            return new ChoiceState(name, {
+                question: $("We are sorry for your loss. Would you like to receive a small set of free messages from MomConnect that could help you in this difficult time?"),
+                choices: [
+                    new Choice("yes", $("Yes")),
+                    new Choice("no", $("No"))
+                ],
+                next: function(choice) {
+                    if (choice.value === "yes") {
+                        var pmtct_loss_switch = {
+                            "registrant_id": self.im.user.answers.identity.id,
+                            "action": "pmtct_loss_switch",
+                            "data": {
+                                "reason": self.im.user.answers.state_optout_reason_menu
+                            }
+                        };
+                        return Q.all([
+                            hub.create_change(pmtct_loss_switch),
+                        ])
+                        .then(function() {
+                            return "state_end_loss_optin";
+                        });
+                    } else {
+                        var pmtct_loss_optout = {
+                            "registrant_id": self.im.user.answers.identity.id,
+                            "action": "pmtct_loss_optout",
+                            "data": {
+                                "reason": self.im.user.answers.state_optout_reason_menu
+                            }
+                        };
+                        var optout_info = {
+                            optout_type: "stop",  // default to "stop"
+                            identity: self.im.user.answers.identity.id,
+                            reason: self.im.user.answers.state_optout_reason_menu,  // default to "unknown"
+                            address_type: "msisdn",  // default to 'msisdn'
+                            address: self.im.user.answers.msisdn,
+                            request_source: "ussd_pmtct",
+                            requestor_source_id: self.im.config.testing_message_id || self.im.msg.message_id,
+                        };
+
+                        return Q
+                        .all([
+                            hub.create_change(pmtct_loss_optout),
+                            is.optout(optout_info),
```

Comment:
remove the `is.optout` here - we're now only opting identities out if they SMS in - otherwise it's an unsubscribe handled in the backend only.